site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -142,6 +142,37 @@ chrome.sidePanel
 With `openPanelOnActionClick()` enabled, you can also open the side panel using a keyboard shortcut by specifying an [action command][action-commands] in the manifest as featured in the [Tab-specific side panel][sample-sp-google] sample.
 {% endAside %}
 
+### Open the side panel with any user interaction {: #user-interaction }
+
+Chrome 116 introduces [`sidePanel.open()`][sidepanel-open]. It allows extensions to invoke the side panel through a user interaction, such as an [action click][api-action], a [keyboard shortcut][api-commands], a [context menu][api-menu], or a button click on an extension page or [content script][doc-cs]. For a complete demo, see the [Open Side Panel][sample-sp-open] sample extension.
+
+This example shows how to open a side panel in all the tabs of the current window when the user clicks on a context menu:
+
+{% Label %}service-worker.js:{% endLabel %}
+
+```js/11
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.contextMenus.create({
+    id: 'openSidePanel',
+    title: 'Open side panel',
+    contexts: ['all']
+  });
+});
+
+chrome.contextMenus.onClicked.addListener((info, tab) => {
+  if (info.menuItemId === 'openSidePanel') {
+    // This will open the panel in all the pages on the current window.
+    chrome.sidePanel.open({ windowId: tab.windowId });
+  }
+});
+```
+
+When using [`sidePanel.open()`][sidepanel-open], you must choose the context in which it should open. Use [`windowId`][sidepanel-windowid] to open a global side panel in the specified window. Alternatively, set the [`tabId`][sidepanel-tabid] to open the side panel only on a specific tab.
+
+{% Aside 'important' %}
```

Comment:
@sebastianbenz I'm debating if this would fit better in the ## Overview section 🧐

Reply:
You mean the previous or the following paragraph? I think it makes sense here as this is particularly relevant for sidepanel.open.

Reply:
I meant the aside. I'll leave it where it's al